tests/unittests/import_export_tests.py

```diff
@@ -2104,17 +2104,19 @@ def _make_dataset(self):
     def test_csv_dataset(self):
         dataset = self._make_dataset()
 
-        # @todo uncomment once CSVDatasetImporter is implemented
-        """
-        field_parsers = {
-            "weather": lambda value: (
-                fo.Classification(label=value) if value is not None else None
-            ),
-            "filepath": None,  # default parser
-            "tags": lambda value: value.strip("").split(","),
-            "float_field": lambda value: float(value),
+        export_fields = {
+            "filepath": "filepath",
+            "tags": "tags",
+            "float_field": "float_field",
+            "weather.label": "weather",
```

Comment:
Uses the `export_fields=dict` syntax to be able to export a nested field value and then reconstruct it in the same format.

For future work: we should probably add a way for `CSVDatasetImporter` to handle columns names that contain `.`.